package.json

```diff
@@ -74,6 +74,7 @@
     "esm": "^3.2.25",
     "glob": "^7.1.6",
     "localforage": "^1.9.0",
+    "material-design-icons": "^3.0.1",
```

Comment:
We can revisit this, but I do wonder how much this will add to the build size?

Reply:
I had thought it would be small, because the [docs](https://google.github.io/material-design-icons/) say that "The icon font weighs in at only 42KB in its smallest woff2 format and 56KB in standard woff format." But checking it with `du -sh node_modules/material-design-icons/` I get that it's 356 megabytes! Not sure if that necessarily correlates with the build size.